content/blog/2017/0.1-canary/index.md

```diff
@@ -2,6 +2,7 @@
 title: Canary Deployments using Istio
 description: Using Istio to create autoscaled canary deployments
 publishdate: 2017-06-14
+lastmod: 2018-07-26
```

Comment:
We actually get the last modified date from GitHub directly, so this is not needed.